app/lib/submission_builder/ty2024/states/nj/documents/nj1040.rb

```diff
@@ -86,7 +92,7 @@ def intake
             end
 
             def calculated_fields
-              @nj1040_fields ||= intake.tax_calculator.calculate
+              @nj1040_fields ||= @submission.data_source.tax_calculator.calculate
```

Comment:
[dust, very small particles] why access directly instead of using the method above for consistency?

Reply:
oversight! didn't realize `intake` referred to the method above doing the same thing!